code/data_cleaning/format_data.py

```diff
@@ -0,0 +1,49 @@
+import pandas as pd
+import numpy as np
+from keras.utils import np_utils 
```

Comment:
also, there is some art to imports - this is actually an important part of PEP8, [link](https://www.python.org/dev/peps/pep-0008/#imports) - some people pick alphabetical as well.